### PR TITLE
MOC 5 cards

### PR DESCRIPTION
**Pull request by Simisays**
Don't know if it's useful to assign deckhints to plane cards ?

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/sidar_jabari_of_zhalfir.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Name:Sidar Jabari of Zhalfir
+ManaCost:1 W U B
+Types:Legendary Creature Human Knight
+PT:4/3
+T:Mode$ AttackersDeclared | ValidAttackers$ Card.Knight+YouCtrl | TriggerZones$ Command | PresentZone$ Command | Execute$ TrigLoot | TriggerDescription$ Eminence — Whenever you attack with one or more Knights, if CARDNAME is in the command zone or on the battlefield, draw a card, then discard a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sidar_jabari_of_zhalfir.txt:5]: Both missing `IsPresent$ Card.StrictlySelf`

File: forge-gui/res/cardsfolder/upcoming/sidar_jabari_of_zhalfir.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+T:Mode$ AttackersDeclared | ValidAttackers$ Card.Knight+YouCtrl | TriggerZones$ Battlefield | PresentZone$ Battlefield | Execute$ TrigLoot | Secondary$ True | TriggerDescription$ Eminence — Whenever you attack with one or more Knights, if CARDNAME is in the command zone or on the battlefield, draw a card, then discard a card.
+SVar:TrigLoot:DB$ Draw | Defined$ You | NumCards$ 1 | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | Defined$ You | NumCards$ 1 | Mode$ TgtChoose
+K:Flying
+K:First Strike
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigReturn | TriggerZones$ Battlefield | TriggerDescription$ Whenever NICKNAME deals combat damage to a player, return target Knight creature card from your graveyard to the battlefield.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sidar_jabari_of_zhalfir.txt:11]: NICKNAME parsing will fail here, need to think on that

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/sidar_jabari_of_zhalfir.txt:11]: I guess CARDNAME would do for now ?

File: forge-gui/res/tokenscripts/r_0_0_a_gremlin.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Name:Gremlin Artifact Token
```
Comment (Northmoc) [inline forge-gui/res/tokenscripts/r_0_0_a_gremlin.txt:1]: I think this should just be Gremlin Token

File: forge-gui/res/cardsfolder/upcoming/sidar_jabari_of_zhalfir.txt [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+T:Mode$ AttackersDeclared | ValidAttackers$ Card.Knight+YouCtrl | TriggerZones$ Battlefield | PresentZone$ Battlefield | IsPresent$ Card.StrictlySelf | Execute$ TrigLoot | Secondary$ True | TriggerDescription$ Eminence — Whenever you attack with one or more Knights, if CARDNAME is in the command zone or on the battlefield, draw a card, then discard a card.
+SVar:TrigLoot:DB$ Draw | Defined$ You | NumCards$ 1 | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | Defined$ You | NumCards$ 1 | Mode$ TgtChoose
+K:Flying
+K:First Strike
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigReturn | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, return target Knight creature card from your graveyard to the battlefield.
+SVar:TrigReturn:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | ChangeNum$ 1 | ValidTgts$ Creature.YouOwn+Knight
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/sidar_jabari_of_zhalfir.txt:12]: This should have TgtPrompt

File: forge-gui/res/cardsfolder/upcoming/sidar_jabari_of_zhalfir.txt (hunk 2 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/sidar_jabari_of_zhalfir.txt]: yea probably better

File: forge-gui/res/cardsfolder/upcoming/sidar_jabari_of_zhalfir.txt [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+T:Mode$ AttackersDeclared | ValidAttackers$ Card.Knight+YouCtrl | TriggerZones$ Battlefield | PresentZone$ Battlefield | IsPresent$ Card.StrictlySelf | Execute$ TrigLoot | Secondary$ True | TriggerDescription$ Eminence — Whenever you attack with one or more Knights, if CARDNAME is in the command zone or on the battlefield, draw a card, then discard a card.
+SVar:TrigLoot:DB$ Draw | Defined$ You | NumCards$ 1 | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | Defined$ You | NumCards$ 1 | Mode$ TgtChoose
+K:Flying
+K:First Strike
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigReturn | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, return target Knight creature card from your graveyard to the battlefield.
+SVar:TrigReturn:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | TgtPrompt$ Select target Knight creature card | ChangeNum$ 1 | ValidTgts$ Creature.YouOwn+Knight
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/sidar_jabari_of_zhalfir.txt:12]: prompt should probably say "in your graveyard" or something like that

**Review by tool4ever — APPROVED** [review]

**Pull request merged**